### PR TITLE
Swap undo/redo handlers from electron accelerators to handler hooks

### PR DESCRIPTION
**Pull request by joeyballentine**

### CODE REVIEW

File: src/renderer/components/HistoryProvider.tsx [hunk 1]
```diff
@@ -122,6 +122,24 @@ export const HistoryProvider = ({ children }: React.PropsWithChildren<unknown>):
...
+        'ctrl+z, cmd+z',
+        () => {
+            historyObj.history = historyObj.history.undo();
+            apply(historyObj.history.current);
+        },
+        [apply]
+    );
+
+    useHotkeys(
+        'ctrl+y, cmd+y',
```
Comment (RunDevelopment) [inline src/renderer/components/HistoryProvider.tsx:135]: Please add 'ctrl+shift+z, cmd+shift+z' to this.

File: src/renderer/components/HistoryProvider.tsx [hunk 2]
```diff
@@ -100,7 +100,7 @@ export const HistoryProvider = ({ children }: React.PropsWithChildren<unknown>):
 
         const id = setTimeout(() => {
             historyObj.history = historyObj.history.commit([getNodes(), getEdges()]);
-        }, 500);
+        }, 250);
```
Comment (RunDevelopment) [inline src/renderer/components/HistoryProvider.tsx:103]: Why?

File: src/renderer/components/HistoryProvider.tsx (hunk 1 above)
Comment (joeyballentine) [reply src/renderer/components/HistoryProvider.tsx:135]: oh yeah, duh

File: src/renderer/components/HistoryProvider.tsx (hunk 2 above)
Comment (joeyballentine) [reply src/renderer/components/HistoryProvider.tsx:103]: Because it's not needed anymore. I only increased it to 500 to try to fix the number input issues

**Review by RunDevelopment — APPROVED** [review]

**Pull request merged**